_components/search-bar.md

```diff
@@ -9,11 +9,13 @@ lead: Intro text on what is included in this section and how to use it. No more
   <div class="usa-grid">
     <div class="usa-width-one-half">
       <form class="usa-search usa-search-big">
-        <label for="search-field-big">Search Big</label>
-        <input type="search" id="search-field-big">
-        <button type="submit">
-          <span class="usa-search-submit-text">Search</span>
-        </button>
+        <div role="search">
```

Comment:
@msecret my view is that putting it inside the form tag is better practice bc it won't get messed up as easily. The wrapper div around it is more for layout even though you could technically add it there. I think the cost of having an extra div on the inside search form tag is less than the cost of someone missing it / messing it up bc they remove the wrapping grid div. I vote to leave it as is.